examples/emu3.5/conf/serve/emu3p5.py

```diff
@@ -0,0 +1,227 @@
+        # "model": "/path/to/models/Emu3.5-Image",   # https://modelscope.cn/models/BAAI/Emu3.5-Image/files
+        "tokenizer": "/path/to/src/tokenizer_emu3_ibq",  # https://github.com/baaivision/Emu3.5/src/tokenizer_emu3_ibp
+        "vq_model": "/path/to/models/Emu3.5-VisionTokenizer",  # https://www.modelscope.cn/models/BAAI/Emu3.5-VisionTokenizer/files
+        "trust_remote_code": True,
+        "tensor_parallel_size": 2,
+        "gpu_memory_utilization": 0.7,
+        "disable_log_stats": False,
+        "enable_chunked_prefill": False,
+        "enable_prefix_caching": False,
+        "seed": 42,
+        "dtype": "auto",
+    },
+    "sampling": {
+        "detokenize": False,
+        "guidance_scale": 3.0,
+        "text_top_k": 1024,
+        "text_top_p": 0.9,
+        "text_temperature": 1.0,
+        "image_top_k": 5120,
+        "image_top_p": 1.0,
+        "image_temperature": 1.0,
+        "top_k": 131072,
+        "top_p": 1.0,
+        "temperature": 1.0,
+        "max_tokens": 5120,
+    },
+}
+
+
+class Emu3_5InferencePipeline:
+
+    def __init__(self, config: Dict[str, Any] = DEFAULT_CONFIG):
+
+        self.cfg = config
+        self.processor: Optional[Emu3p5Processor] = None
+        self.llm: Optional[LLM] = None
+
+        self.task_type = self.cfg["task_type"]
+        assert self.task_type in [
+            "t2i",
+            "x2i",
+            "story",
+            "howto",
+        ], f"Unsupported task_type: {self.task_type}. Options: 't2i', 'x2i', 'story', and 'howto'."
+
+        self.ratio = self.cfg["ratio"]
+
+        self._load_models()
+
+    def _load_models(self):
+
+        cfg = self.cfg
+        llm_cfg = cfg.get("llm", {})
+
+        tokenizer_path = llm_cfg.get("tokenizer", None)
+        vq_model_path = llm_cfg.pop("vq_model", None)
+        assert (
+            tokenizer_path and vq_model_path
+        ), "Please set the tokenzier and vq_model in llm config."
+
+        image_area = cfg["image_area"]
+
+        try:
+            self.processor = Emu3p5Processor(
+                task_type=self.task_type,
+                tokenizer_path=tokenizer_path,
+                vq_model_path=vq_model_path,
+                image_area=image_area,
+                ratio=self.ratio,
+            )
+
+            if self.task_type in ["t2i", "x2i"]:
+                self.processor.stop_token_id = self.processor.special_token_ids["EOI"]
+            else:
+                self.processor.stop_token_id = self.processor.special_token_ids["EOS"]
+
+            logger.info(
+                f"Emu3p5Processor initialized for task: {self.task_type} (Ratio: {self.ratio})."
+            )
+        except Exception as e:
+            logger.error(f"Failed to initialize Emu3p5Processor: {e}")
+            sys.exit(1)
+
+        try:
+            self.llm = LLM(
+                **llm_cfg,
+                max_num_batched_tokens=26000,
+                max_num_seqs=2,
```

Comment:
Why set  these two hyperparameter values ​to 26000 and 2?

Reply:
keep the same as inference version, which means: 4k * 3 (image num) * 2 (req num) +1k (text prompt num)